kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/dhcp/DhcpServerManager.java

```diff
@@ -87,6 +98,14 @@ public boolean enable(String interfaceName) throws KuraException {
         return false;
     }
 
+    private void createLeasesFile(String interfaceName) {
+        try {
+            FileUtils.touch(new File(DhcpServerManager.getLeasesFilename(interfaceName)));
+        } catch (IOException e) {
+            logger.error("Cannot create DHCP server leases file", e);
```

Comment:
Shall we throw a `KuraException` here to prevent starting the interface without leases file?